### PR TITLE
[FEAT] Use project icons

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

**Review by tbouffard — CHANGES_REQUESTED** [review]
We have decided to remove the public folder from the repository as it only contains generated content. See https://github.com/process-analytics/process-analytics.dev/pull/9#discussion_r654224742
This is still a work in progress

**Review by tbouffard — APPROVED** [review]
I am not sure we need both the favicon and the logo files

**Comment by csouchet** [comment]
> I am not sure we need both the favicon and the logo files

I'm not sure too, but I don't now how Gatsby generates all the icon. It generates some icons bigger than the original (the quality is bad). So, maybe we can change that later, when we have a bigger original icon.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+    <p align="center">
+      <a href="https://github.com/process-analytics/process-analytics.dev">
+        <img alt="Process Analytics Logo" src="src/images/logo_64x64.png" width="64" />
+      </a>
+    </p>
+    <p align="center">
+        <a href="CONTRIBUTING.md">
+          <img alt="PRs Welcome" src="https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square"> 
+        </a> 
+        <a href="CODE_OF_CONDUCT.md">
```
[the remark this answers is not in the document]
Comment (csouchet) [reply README.md:11]: I badly merge the conflicts.
I didn't see in a previous PR that I remove the logo 😞

**Comment by tbouffard** [comment]
about the logo, we have it in svg so we can create png, ico, ... files at any resolution.
[Process-Analytics-logo_svg.zip](https://github.com/process-analytics/process-analytics.dev/files/6677905/Process-Analytics-logo_svg.zip)

**Comment by csouchet** [comment]
I did some tests with SVG image, it didn't work. I don't know why.
I try again why only SVG logo & removing the other images. And it works ^^

The plugin who generates the icons: https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/

**Comment by tbouffard** [comment]
ok let's start with the single svg logo

**Pull request merged**